2.0.9 overrides adds a node binary to the PATH in npx calls

The 2.0.9 point release installs a `node` binary in the `node_modules/.bin`. In the event that your project adds `node_modules/.bin` to the current PATH, that `node` will take precedence over the system node. In my case I use `npx` which does this PATH management automatically.

This is due to the [commit](https://github.com/cure53/DOMPurify/commit/b26420eef1b10cb8116ca1b774da41896a930d34) that adds the `node` npm package as a dependency. Is it an absolute requirement or can it be installed as a dev dependency?

Example of this unexpected behavior:

```
~/dev/testpackage
❯ npm install dompurify@2.0.7
+ dompurify@2.0.7
added 1 package from 1 contributor and audited 1 package in 0.762s
found 0 vulnerabilities


~/dev/testpackage took 2s
❯ npx node -v
v10.18.0

~/dev/testpackage
❯ npm install dompurify@2.0.9

> node@13.13.0 preinstall /Users/dustym/dev/testpackage/node_modules/node
> node installArchSpecificPackage

+ node-darwin-x64@13.13.0
added 1 package in 5.234s
found 0 vulnerabilities

+ dompurify@2.0.9
added 3 packages from 2 contributors, updated 1 package and audited 4 packages in 7.376s
found 0 vulnerabilities


~/dev/testpackage took 8s
❯ npx node -v
v13.13.0
```

As an additional datapoint, this is also causing install to fail when using yarn 2 ("berry") with nodeModules linker. The downstream dependency in my case is `swagger-ui-react`:

```
➤ YN0000: ┌ Link step
➤ YN0007: │ node@npm:13.13.0 must be built because it never did before or the last one failed
➤ YN0009: │ node@npm:13.13.0 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-2c48a784/build.log)
➤ YN0009: │ node@npm:13.13.0 couldn't be built successfully (exit code 1, logs can be found here: /tmp/xfs-2c48a784/build.log)
➤ YN0000: └ Completed in 12.34s
➤ YN0000: Failed with errors in 58.33s

root@e8ea159cdb2a:/src/js/cloud# yarn why node
└─ dompurify@npm:2.0.9
   └─ node@npm:13.13.0 (via npm:^13.13.0)

root@e8ea159cdb2a:/src/js/cloud# yarn why dompurify
├─ swagger-ui-react@npm:3.25.0
│  └─ dompurify@npm:2.0.9 (via npm:^2.0.7)
│
└─ swagger-ui-react@npm:3.25.0 [d119c]
   └─ dompurify@npm:2.0.9 (via npm:^2.0.7)

root@e8ea159cdb2a:/src/js/cloud# cat /tmp/xfs-2c48a784/build.log
# This file contains the result of Yarn building a package (node@npm:13.13.0)
# Script name: preinstall

internal/modules/cjs/loader.js:979
  throw err;
  ^

Error: Cannot find module '/src/js/cloud/node_modules/node/bin/node'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:976:15)
    at Function.Module._load (internal/modules/cjs/loader.js:859:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

--- a/package.json
+++ b/package.json
@@ -60,10 +60,10 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-external-helpers": "7.8.3",
     "babel": "^6.23.0",
     "babel-core": "^6.26.3",
     "babel-preset-env": "^1.7.0",
-    "@babel/plugin-external-helpers": "7.8.3",
     "cross-env": "^7.0.2",
     "eslint-config-prettier": "^6.10.1",
     "eslint-plugin-prettier": "^3.1.3",
@@ -82,7 +82,9 @@
     "lodash.isarray": "^4.0.0",
     "lodash.sample": "^4.2.1",
     "minimist": "^1.2.5",
+    "node": "^13.13.0",
     "npm-run-all": "^4.1.5",
+    "nvm": "0.0.4",
     "pre-commit": "^1.1.2",
     "prettier": "^2.0.4",
     "qunit-parameterize": "^0.4.0",
@@ -131,8 +133,5 @@
     "url": "https://github.com/cure53/DOMPurify/issues"
   },
   "homepage": "https://github.com/cure53/DOMPurify",
-  "dependencies": {
-    "node": "^13.13.0",
-    "nvm": "0.0.4"
-  }
+  "dependencies": {}
 }

